frontend/components/common/formatters/AnimatedNumber.vue

```diff
@@ -32,8 +33,18 @@ export default Vue.extend({
             type: Number,
             default: undefined,
         },
+        formatted: {
+            type: Boolean,
+            default: false,
+        },
```

Comment:
If we want to introduce this feature, why not setting it on by default? Then, please rename it into `disableThousandSeparators`, since `formatted` doesn't really tell what it does (the number is already formatted anyway)